src/serialization/sb2.js

```diff
@@ -208,6 +209,104 @@ const globalBroadcastMsgStateGenerator = (function () {
     };
 }());
 
+/**
+ * Parse a single monitor object and create all its in-memory VM objects.
+ * @param {!object} object - From-JSON "Scratch object"
```

Comment:
We can provide more info here. The `object` param should represent a monitor and not a sprite (which also qualifies as a 'Scratch object').